apps/health_campaign_field_worker_app/lib/data/repositories/api_interceptors.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:dio/dio.dart';
+
+import '../../models/request_info/request_info_model.dart';
+
+class ApiInterceptors extends Interceptor {
+  @override
+  Future<dynamic> onRequest(
+    RequestOptions options,
+    RequestInterceptorHandler handler,
+  ) async {
+    options.data = {
+      ...options.data,
+      "RequestInfo": const RequestInfoModel(
```

Comment:
Please read the properties of RequestInfo from a configuration instead of hard-coding.

Reply:
@egov-joy  Moved to constants